test/testutils.jl

```diff
@@ -40,7 +41,12 @@ function test_distr(distr::DiscreteUnivariateDistribution, n::Int;
 
     test_stats(distr, vs)
     test_samples(distr, n)
-    test_samples(distr, n, rng=MersenneTwister())
+    test_samples(distr, n, rng=rng)
+    if test_scalar_rand
+        xs = test_samples(distr, n; call_scalar = true)
+        xs = test_samples(distr, n, rng=rng, call_scalar = true)
+    end
+
```

Comment:
I think we can just revert to
```suggestion
    test_samples(distr, n, rng=MersenneTwister())
```